prompterator/models/openai_models.py

```diff
@@ -89,42 +88,102 @@ def __init__(self):
 
         super().__init__()
 
+    @staticmethod
+    def get_function_calling_tooling_name(json_schema):
+        function = json_schema.copy()
+        return function.pop("title")
```

Comment:
Maybe this would also do the job? 🙂 It fails when the `title` key isn't found -- same as the `.pop` call (when `pop`'s second arg isn't provided).
```suggestion
        return json_schema["title"]
```